src/main/java/eu/dissco/core/digitalmediaprocessor/domain/DigitalMediaKey.java

```diff
@@ -0,0 +1,5 @@
+package eu.dissco.core.digitalmediaprocessor.domain;
+
+public record DigitalMediaKey(String digitalSpecimenId, String mediaUrl) {
```

Comment:
You know, it might be useful to have these in one field? something like, ods:normalisedMediaId? I'm thinking about combining these fields in the handle api, so it would be nice to be consistent. beyond scope rn though

Reply:
Good point, we added this one to the refinement topics